frontend/src/components/HostMetricsDetail/HostMetricsLogs/HostMetricLogsDetailedView.tsx

```diff
@@ -0,0 +1,137 @@
+import './HostMetricLogs.styles.scss';
+
+import QueryBuilderSearch from 'container/QueryBuilder/filters/QueryBuilderSearch';
+import DateTimeSelectionV2 from 'container/TopNav/DateTimeSelectionV2';
+import {
+	CustomTimeType,
+	Time,
+} from 'container/TopNav/DateTimeSelectionV2/config';
+import { useQueryBuilder } from 'hooks/queryBuilder/useQueryBuilder';
+import GetMinMax from 'lib/getMinMax';
+import { memo, useCallback, useMemo, useState } from 'react';
+import { DataTypes } from 'types/api/queryBuilder/queryAutocompleteResponse';
+import { IBuilderQuery } from 'types/api/queryBuilder/queryBuilderData';
+import { DataSource } from 'types/common/queryBuilder';
+import { v4 as uuidv4 } from 'uuid';
+
+import HostMetricsLogs from './HostMetricsLogs';
+
+interface Props {
+	hostName: string;
+	timeRange: {
+		startTime: number;
+		endTime: number;
+	};
+	isModalTimeSelection: boolean;
+}
+
+function HostMetricLogsDetailedView({
+	hostName,
+	timeRange,
+	isModalTimeSelection,
+}: Props): JSX.Element {
+	const [modalTimeRange, setModalTimeRange] = useState(timeRange);
+	const [, setSelectedInterval] = useState<Time>('5m');
+	const [filters, setFilters] = useState<IBuilderQuery['filters']>({
+		op: 'AND',
+		items: [],
+	});
+	const { currentQuery } = useQueryBuilder();
+	const updatedCurrentQuery = useMemo(
+		() => ({
+			...currentQuery,
+			builder: {
+				...currentQuery.builder,
+				queryData: [
+					{
+						...currentQuery.builder.queryData[0],
+						dataSource: DataSource.LOGS,
+						aggregateOperator: 'noop',
+						aggregateAttribute: {
+							...currentQuery.builder.queryData[0].aggregateAttribute,
+						},
+					},
+				],
+			},
+		}),
+		[currentQuery],
+	);
+
+	const query = updatedCurrentQuery?.builder?.queryData[0] || null;
+
+	// const { handleChangeQueryData } = useQueryOperations({
+	//     index: 0,
+	//     query,
+	//     isListViewPanel: true,
+	//     entityVersion: '',
+	// });
```

Comment:
commented code

Reply:
@SagarRajput-7 I have kept this commented code because we might have to use this code to redirect the user to logs page. I will confirm on this otherwise I will remove the commented code.